esp/src/src-react/components/WorkunitDetails.tsx

```diff
@@ -22,12 +22,14 @@ interface WorkunitDetailsProps {
     wuid: string;
     tab?: string;
     state?: string;
+    queryParams?: { [key: string]: string };
```

Comment:
My intention with state, is that its additional "Tab" state that is a general placeholder for any additional info that needs to be passed through to the individual tabs.
So it may be better to rename "state" to "tabState" and remove queryParams (as it can reuse the tabState param) - then keep it as a string in the router and then each tab page can set and parse it as appropriate?